public/js/ChatController.js

```diff
@@ -65,15 +67,17 @@
                 });
         }
 
-        function guestLogin() {
-            RequestService.postGuestLogin($scope.nameInputBox)
-                .then(function (response) {
-                    $scope.loadUserInfo();
-                    $scope.errorText = "";
-                }, function (response) {
-                    $scope.errorText =
-                        "Failed to login as guest : " + response.status + " - " + response.statusText;
-                });
+        function guestLogin(name, avatarURL) {
+            if (name !== "") {
```

Comment:
Same issue that Steven pointed out. Thinking about using !name or another alternative.